src/lib/Components/Bidding/Screens/ConfirmBid/PriceSummary.tsx

```diff
@@ -0,0 +1,95 @@
+import { Box, Flex, Sans, Serif } from "@artsy/palette"
+import React from "react"
+import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
+
+import { Bid } from "lib/Components/Bidding/types"
+import { defaultEnvironment } from "lib/relay/createEnvironment"
+import renderWithLoadProgress from "lib/utils/renderWithLoadProgress"
+
+import { PriceSummary_calculatedCost } from "__generated__/PriceSummary_calculatedCost.graphql"
+import { PriceSummaryQuery, PriceSummaryQueryVariables } from "__generated__/PriceSummaryQuery.graphql"
+
+interface PriceSummaryViewProps {
+  calculatedCost: PriceSummary_calculatedCost
+  bid: Bid
+}
+
+const _PriceSummary = ({ bid, calculatedCost }: PriceSummaryViewProps) => (
+  <Box mt={4} mx={4}>
+    <Serif mb={1} size="4" weight="semibold" color="black100">
+      Summary
+    </Serif>
+
+    <Flex mb={1} flexDirection="row" justifyContent="space-between">
+      <Sans size="3" color="black100">
+        Your max bid
+      </Sans>
+      <Sans size="3" color="black100">
+        {`${bid.display}.00`}
+      </Sans>
+    </Flex>
+
+    <Flex mb={1} flexDirection="row" justifyContent="space-between">
+      <Sans size="3" color="black100">
+        Buyer’s premium
+      </Sans>
+      <Sans size="3" color="black100">
+        {calculatedCost.buyersPremium.display}
+      </Sans>
+    </Flex>
+
+    <Flex mb={1} flexDirection="row" justifyContent="space-between">
+      <Sans size="3" color="black100">
+        Subtotal
+      </Sans>
+      <Sans size="3" color="black100">
+        {calculatedCost.subtotal.display}
+      </Sans>
+    </Flex>
+
+    <Sans size="3" color="black60">
+      Plus any applicable shipping, taxes, and fees.
+    </Sans>
+  </Box>
+)
+
+const PriceSummaryFragmentContainer = createFragmentContainer(_PriceSummary, {
+  calculatedCost: graphql`
+    fragment PriceSummary_calculatedCost on CalculatedCost {
+      buyersPremium {
+        display
+      }
+      subtotal {
+        display
+      }
+    }
+  `,
+})
+
+interface PriceSummaryProps extends Partial<PriceSummaryQueryVariables> {
+  bid: Bid
+}
+
+export const PriceSummary = ({ saleArtworkId, bid }: PriceSummaryProps) => (
+  <QueryRenderer<PriceSummaryQuery>
+    environment={defaultEnvironment}
+    query={graphql`
+      query PriceSummaryQuery($saleArtworkId: ID!, $bidAmountCents: Int!) {
+        node(id: $saleArtworkId) {
+          ... on SaleArtwork {
+            calculatedCost(bidAmountCents: $bidAmountCents) {
+              ...PriceSummary_calculatedCost
+            }
+          }
+        }
+      }
+    `}
+    variables={{
+      saleArtworkId,
+      bidAmountCents: bid.cents,
+    }}
+    render={renderWithLoadProgress(({ node }) => (
```

Comment:
if this doesn't work, should we still keep it around?

Reply:
The only thing that's not working is the animation and the `<Spinner>` component itself is still mountable. I don't think it is worth dropping given that the `renderWithLoadProgress` takes care of [a lot of cases](https://github.com/artsy/emission/blob/6d0e5f50022e81938a29e22f916b1850754d46aa/src/lib/utils/renderWithLoadProgress.tsx#L16-L51) (showing a loader, presenting an error message in case of a failure, sending an error to Sentry).

Reply:
Gotcha, wasn't aware of the other features here. Makes sense to me!